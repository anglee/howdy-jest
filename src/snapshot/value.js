export const text = "hello hello";
export const number = 1234;
export const object = { foo: 'bar'};
export const array = [text, number, object];
export const func = (a) => a * a;