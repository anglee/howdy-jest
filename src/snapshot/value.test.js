import { text, number, object, array, func } from './value';

describe('text', () => {
  it('should match snapshot', () => {
    expect(text).toMatchSnapshot();
  });
});

describe('number', () => {
  it('should match snapshot', () => {
    expect(number).toMatchSnapshot();
  });
});

describe('object', () => {
  it('should match snapshot', () => {
    expect(object).toMatchSnapshot();
  });
});

describe('array', () => {
  it('should match snapshot', () => {
    expect(array).toMatchSnapshot();
  });
});

describe('func', () => {
  it('should match snapshot', () => {
    expect(func).toMatchSnapshot();
  });
  it('should return expected value', () => {
    expect(func(2)).toEqual(4);
  });
});
