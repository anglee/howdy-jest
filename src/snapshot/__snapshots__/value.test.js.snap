exports[`array should match snapshot 1`] = `
Array [
  "hello hello",
  1234,
  Object {
    "foo": "bar"
  }
]
`;

exports[`func should match snapshot 1`] = `[Function func]`;

exports[`number should match snapshot 1`] = `1234`;

exports[`object should match snapshot 1`] = `
Object {
  "foo": "bar"
}
`;

exports[`text should match snapshot 1`] = `"hello hello"`;
